ca/certificate-authority.go

```diff
@@ -125,6 +125,12 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
+	for k, v := range cfsslConfigObj.Signing.Profiles {
+		if len(v.Policies) == 0 {
+			return nil, fmt.Errorf("CFSSL signing profile [%s] contains no policy OIDs", k)
+		}
+	}
+
```

Comment:
This is incorrect.  We shouldn't force any user of this stack to have certificate policy identifiers.  Just check that what is configured gets set.